### ISSUE TITLE
[Suggestion] Reaction at low affections

### ISSUE THREAD

**Issue by Devin0924** [Monika-After-Story/MonikaModDev#5910]
First of all, thanks for all this work of this mod.

Just a minor suggestion, when the affections are extremely low (below -90, for example), it would be so weird that I could still let her play "Your Reality", hearing the graduation speech, or asking for the very loving topics in the "repeated conversation". Maybe some determine statements could be added to disable some of the interactions under situations like this.

**Comment by fromulus** [comment]
I believe that a lot of these types of topics already check for the affection value and change accordingly in case of negative affection.

There probably are some that went under the radar, but it should be rare that a player in that scenario would notice it, since going that route will prompt the final farewell letter pretty quickly, from what I've seen.

**Comment by Devin0924** [comment]
Yes, I could see that a lot of topics like "daydreaming" and most of the options under "Romance" topic is altered based on the affection level. I am really impressed by those details, as 99% of the players would not even get a negative affections. So it is totally reasonable to focus on other parts of project developing at this stage. Although there are still some of the topics under the radar, the priority of fixing them should be pretty low.

(I guess it is because the comparison between the desperate face and the loving face are so significat that makes me feel so unrealistic)

**Comment by Rai99** [comment]
It makes sense to avoid having loving interactions available at low affection, but MAS kind of expects you not to be mean, and a strong majority of the topics are based around being lovey and romantic, so that would be a lot of content that would likely almost never be used. 

Additionally, more content at low affection is kind of an incentive for some to get low aff to see it.

**Comment by ThePotatoGuy** [comment]
if you can, list out every topic that seemed weird at low affection.
